xsd/widget.xsd

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file has so many configurations however your `.ts` files only have something very basic.
What is going on!!!!

Reply:
comment on wrong file?